### PR TITLE
tidb: add description about `TIDB_CHECK_CONSTRAINTS`

### PR DESCRIPTION
**Pull request by CbcWestwolf**
### What is changed, added or deleted? (Required)

Add description about `TIDB_CHECK_CONSTRAINTS`

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v8.4 (TiDB 8.4 versions)
- [ ] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/tidb/pull/48313

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by lcwangchao — APPROVED** [review]

File: information-schema/information-schema-tidb-check-constraints.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+title: TIDB_CHECK_CONSTRAINTS
+summary: Learn the `TIDB_CHECK_CONSTRAINTS` INFORMATION_SCHEMA table.
+---
+
+# TIDB\_CHECK\_CONSTRAINTS
+
+The `TIDB_CHECK_CONSTRAINTS` table provides information about [`CHECK` constraints](/constraints.md#check) on tables. Besides the columns in [`CHECK_CONSTRAINTS`](/information-schema/information-schema-check-constraints.md), `TIDB_CHECK_CONSTRAINTS` provides additional table name and table id where the `CHECK` constraints defined.
```
Comment (hfxsd) [inline information-schema/information-schema-tidb-check-constraints.md:8]:
```suggestion
The `TIDB_CHECK_CONSTRAINTS` table provides information about [`CHECK` constraints](/constraints.md#check) on tables. Besides the columns in [`CHECK_CONSTRAINTS`](/information-schema/information-schema-check-constraints.md), `TIDB_CHECK_CONSTRAINTS` provides additional the table name and table ID that define the `CHECK` constraint.
```

File: information-schema/information-schema-tidb-check-constraints.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
+---
+title: TIDB_CHECK_CONSTRAINTS
+summary: Learn the `TIDB_CHECK_CONSTRAINTS` INFORMATION_SCHEMA table.
+---
+
+# TIDB\_CHECK\_CONSTRAINTS
+
+The `TIDB_CHECK_CONSTRAINTS` table provides information about [`CHECK` constraints](/constraints.md#check) on tables. Besides the columns in [`CHECK_CONSTRAINTS`](/information-schema/information-schema-check-constraints.md), `TIDB_CHECK_CONSTRAINTS` provides additional the table name and table ID that define the `CHECK` constraint.
```
Comment (hfxsd) [inline information-schema/information-schema-tidb-check-constraints.md:8]:
```suggestion
The `TIDB_CHECK_CONSTRAINTS` table provides information about [`CHECK` constraints](/constraints.md#check) on tables. Besides the columns in [`CHECK_CONSTRAINTS`](/information-schema/information-schema-check-constraints.md), `TIDB_CHECK_CONSTRAINTS` provides the table name and table ID that define the `CHECK` constraint.
```

File: information-schema/information-schema-tidb-check-constraints.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
+---
+title: TIDB_CHECK_CONSTRAINTS
+summary: Learn the `TIDB_CHECK_CONSTRAINTS` INFORMATION_SCHEMA table.
+---
+
+# TIDB\_CHECK\_CONSTRAINTS
+
+The `TIDB_CHECK_CONSTRAINTS` table provides information about [`CHECK` constraints](/constraints.md#check) on tables. Besides the columns in [`CHECK_CONSTRAINTS`](/information-schema/information-schema-check-constraints.md), `TIDB_CHECK_CONSTRAINTS` provides the table name and table ID that define the `CHECK` constraint.
```
Comment (lilin90) [inline information-schema/information-schema-tidb-check-constraints.md:8]:
```suggestion
The `TIDB_CHECK_CONSTRAINTS` table provides information about [`CHECK` constraints](/constraints.md#check) on tables. In addition to the columns in [`CHECK_CONSTRAINTS`](/information-schema/information-schema-check-constraints.md), `TIDB_CHECK_CONSTRAINTS` provides the name and ID of the table that defines the `CHECK` constraint.
```

File: information-schema/information-schema-tidb-check-constraints.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+```
+
+Fields in the `TIDB_CHECK_CONSTRAINTS` table are described as follows:
+
+* `CONSTRAINT_CATALOG`: The catalog of the constraint, which is always `def`.
+* `CONSTRAINT_SCHEMA`: The schema of the constraint.
+* `CONSTRAINT_NAME`: The name of the constraint.
+* `CHECK_CLAUSE`: The clause of the check constraint.
+* `TABLE_NAME`: The table name of the constraint.
+* `TABLE_ID`: The table id of the constraint.
```
Comment (lilin90) [inline information-schema/information-schema-tidb-check-constraints.md:59]:
```suggestion
* `TABLE_NAME`: The name of the table where the constraint is located.
* `TABLE_ID`: The ID of the table where the constraint is located.
```

**Review by lilin90 — APPROVED** [review]
/lgtm

**Pull request merged**